### ISSUE TITLE
Fieldwork: CP ignores manually set tool offset for special tools 

### ISSUE THREAD

**Issue by Voidheartd** [Courseplay/courseplay#4346]
This is an issue I noticed in fieldwork mode, not sure if it affects other modes.

I noticed that the tool offset horizontal changes when CP is driving versus when it is off. I realized it when I had changed the offset while CP was driving and some time later had problems because that time I changed the offset while CP was off but then found the vehicle was at the other offset instead, so I had to fix the offset while CP was driving.

Currently, I am using the bale wrapper on a fieldwork course. When CP is off, the tool offset horizontal is "---". When I start CP the offset becomes "2.5m (Left)" which is the correct setting. When I stop CP, it goes back to "---". From my testing, it seems like the offset chosen while CP is off has no effect.

This makes it harder to set custom offsets if it has to be done while CP is driving.

**Comment by Tensuko** [comment]
But what happens if you set it to 2.5m (left) and then start CP ? Does it become something else ?
Or stays to 2.5m (left) ?

**Comment by Voidheartd** [comment]
It stays at 2.5m (left) because the 2 numbers are not connected. No matter what I set it to before starting CP, it goes to 2.5m (left) or whatever I set while CP is running.

It does appear that on first loading in, the value that is used while CP is running does show up when CP is not running. This stops as soon as you start CP and then stop it. After that, the disconnect occurs.

**Comment by pops64** [comment]
Also this value is not adjustable while CP is driving ... This is annoying when the default selected offset is not correct.

**Comment by pvaiko** [comment]
This should only happen with a plow.

**Comment by Voidheartd** [comment]
It was happening with baling equipment.

**Comment by pvaiko** [comment]
You have to help me here, I don't understand what the problem is. When running a bale wrapper, with tool offset -- the wrapper is right on the course and I can move it left or right of that by changing the offset. What exactly is the issue here?

**Comment by Voidheartd** [comment]
? I explained this in the beginning. 

"I noticed that the tool offset horizontal changes when CP is driving versus when it is off. I realized it when I had changed the offset while CP was driving and some time later had problems because that time I changed the offset while CP was off but then found the vehicle was at the other offset instead, so I had to fix the offset while CP was driving."

The tool offset value when CP is on/off is desynching. The value I set when CP is off and the value I set when CP is on are completely separate. That means if I want to change the value CP is going to use, I have to do so when it is driving. 

This easily leads to confusion if someone is not aware (as I was) that this is happening. I had changed the offset for one field. I ended up having part of another field done with that offset when I didn't want it because I checked the offset before starting CP (as you would to verify your settings) when the value CP was going to show and use as soon as it was on was something else.

The offset should be the same whether CP is on or off.
